### PR TITLE
updated c++ client instructions

### PR DESCRIPTION
**Pull request by katiemacaulay**
**Overview**
Customer was following getting started client instructions from readthedocs and mentioned that they wish they had read the Readme instead because it has better pictures and information. 

**Changes**
Updated the RTD getting started instructions from client repo instructions

https://monodrive.readthedocs.io/en/update_client_getting_started/cpp_client/cpp_quick_start/

### CODE REVIEW

File: docs/cpp_client/cpp_quick_start.md [hunk 1]
```diff
@@ -20,52 +17,209 @@ Ubuntu 18.04.
...
+- [OpenCV](https://github.com/opencv/opencv/releases/download/4.3.0/opencv-4.3.0-vc14_vc15.exe) Extract to `C:\local\opencv` and add `C:\local\opencv\build\x64\vc15\bin` to your PATH environment variable.
+
+- [Eigen](https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.zip) Extract to `C:\local\Eigen3` and add `C:\local\Eigen3` to your PATH environment variable.
+     - **NOTE**: You will need to move the extracted contents out of the version subfolder (eg `.\eigen-3.3.7\`) into the parent directory, such that `C:\local\Eigen3\Eigen` is a valid directory path.
+
+<div class="img_container">
+    <img class="lg_img" src="../imgs/eigen3_dir.jpeg">
+</div>
+
+#### Figure: Environment Variables
```
Comment (bradleynull) [inline docs/cpp_client/cpp_quick_start.md:46]: For some reason using 4 `#` renders as a top level header on my browser. Suggest re-styling this: 
image

File: docs/cpp_client/cpp_quick_start.md [hunk 2]
```diff
@@ -20,52 +17,209 @@ Ubuntu 18.04.
...
+    - Run the prerequisite setup script from the root repository directory. This will install all dependencies and also build a newer version of Boost:
+```bash
+$ ./util/setup.sh
+```
 
-The packages necessary for building the client can be installed with the 
-following:
+- [VSCode](https://code.visualstudio.com/)
 
+## Ubuntu 20.04 Prerequisites
```
Comment (bradleynull) [inline docs/cpp_client/cpp_quick_start.md:62]: The 18.04 is a lower level heading than the 20.04. Suggest making a single `## Ubuntu` section and sub-headings for 18.04 and 20.04

File: docs/cpp_client/cpp_quick_start.md [hunk 3]
```diff
@@ -20,52 +17,209 @@ Ubuntu 18.04.
...
+    hdrs = [...],
+    deps = [
+        ...,
+        "@monodrive//monodrive/core:monodrive"
+    ]
+)
+```
+
+
+## Installation to system on Ubuntu
```
Comment (bradleynull) [inline docs/cpp_client/cpp_quick_start.md:198]: As far as I know, there is no `install` target for the client. I think this should be removed.

File: docs/cpp_client/cpp_quick_start.md [hunk 4]
```diff
@@ -20,52 +17,209 @@ Ubuntu 18.04.
...
+```cmake
+# add monodrive client library from local repo
+add_subdirectory(path/to/monodrive-client/monodrive mdclient)
+
+# link targets as needed
+target_link_libraries(<mytarget> monodrive)
+```
+
+
+## Installation with Bazel
```
Comment (bradleynull) [inline docs/cpp_client/cpp_quick_start.md:175]: This should be a sub-heading of the Ubuntu section. Its simply showing how to use a separate compiler other than `gcc ` or `g++`

File: docs/cpp_client/cpp_quick_start.md (hunk 3 above)
Comment (bradleynull) [reply docs/cpp_client/cpp_quick_start.md:198]: Are these the instructions for installation with Bazel?

**Review by bradleynull — CHANGES_REQUESTED** [review]
@katiemacaulay A few quick things with headings. Also need to discuss the installation stuff.

File: docs/cpp_client/cpp_quick_start.md [hunk 5]
```diff
@@ -1,71 +1,223 @@
...
-## Guide
-
 The monoDrive C++ Client is Open Source Software for connecting to and 
 configuring the monoDrive Simulator and Scenario Editor. To get started, 
-clone the client from the monoDrive repository:
-
-```bash
-$ git clone git@github.com:monoDriveIO/monodrive-client.git
-```
+contact support@monodrive.io to get access to the monoDrive Client repository,
```
[the remark this answers is not in the document]
Comment (katiemacaulay) [reply docs/cpp_client/cpp_quick_start.md:5]: checked and it is still being forwarded to this email

**Review by bradleynull — APPROVED** [review]
@katiemacaulay Looks good to go! Approved!

**Pull request merged**